editor.Rmd

```diff
@@ -75,10 +116,27 @@ A further tweet _can_ include a link to the software peer review thread.
 Any tweet _can_ include
 
 * hashtags,
-* account names,
+* additional account names,
 
 with the caveats of the next subsections.
 
+### Tagging accounts
+
+Tag the account(s) of all post authors and package authors and reviewers. 
+
+If not already included in their author metadata, you can search for their account but only tag them if
+
+* it is active,
+* it is not anonymous,
+* it is at least partly used professionally by the owner.
+
+When in doubt, use the person's name, or time-permitting, contact the person whose account you'd like to tag.
+
+For a package wrapping a service present on Twitter you can tag that account (e.g. for a tweet about `rredlist` that accesses IUCN Red List you might tag IUCN Twitter account).
+
+When tagging accounts include them in a sentence e.g. "Thanks to `@account1`"/ "As told by `@account2`". 
+We do not use tagging in tweets to ask for attention (i.e. no account names used liked hashtags at the bottom of a tweet).
```

Comment:
```suggestion
We do not use tagging in tweets to ask for attention (i.e. no account names used liked hashtags at the bottom of a tweet) because it could be viewed as spamming mentioned accounts, and because it creates visual clutter in the tweet.
```